packages/observe-sequence/observe_sequence_tests.js

```diff
@@ -93,6 +93,37 @@ runOneObserveSequenceTestCase = function (test, sequenceFunc,
              compress(EJSON.stringify(expectedCallbacks, {canonical: true, indent: true})));
 };
 
+// This creates an object subclassed from Array
+const hasProp = {}.hasOwnProperty;
+const extend = function(child, parent) {
+  function ctor() {
+    this.constructor = child;
+  }
+  for(let key in parent) {
+    if(hasProp.call(parent, key)) {
+      child[key] = parent[key];
+      console.log('key: ' + key);
+    }
+  }
+  ctor.prototype = parent.prototype;
+  child.prototype = new ctor();
+  child.__super__ = parent.prototype;
+  return child;
+};
+const ArraySubclass = (function (superClass) {
+  extend(ArraySubclass, superClass);
+
+  function ArraySubclass() {
+    if (arguments.length > 0) {
+      const items = this.slice.call(arguments, 0);
+      this.splice.apply(this, [0, 0].concat(this.slice.call(items)));
+    }
+  }
+
+  return ArraySubclass;
+
+})(Array);
+
```

Comment:
Thanks @abernix,  I just tried it.

`vm` is not a global
I cannot `require('vm')` - needs browserify package
I cannot `required('vm')` - defined in Utils package
I cannot `Npm.required('vm')` - ??
I cannot `import vm from 'vm';`
I cannot `import {vm} from 'vm';`

Do you know how a test file can load a module?  I do not see any `require`-like statements in the test files unless they are in `node_modules` installed under the `.npm` directory.